lib/improver/tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -625,6 +661,40 @@ def test_day_night(self):
         self.assertArrayEqual(result.data,
                               expected_wxcode)
 
+    def test_no_lightning(self):
+        """Test process returns right values if no lightning. """
+        plugin = WeatherSymbols()
+        result = plugin.process(self.cubes_no_lightning)
+        msg = '<WeatherSymbols tree=high_resolution_without_lightning>'
+        self.assertEqual(str(plugin), msg)
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertArrayEqual(result.attributes['weather_code'], self.wxcode)
+        self.assertEqual(result.attributes['weather_code_meaning'],
+                         self.wxmeaning)
+        expected_wxcode = np.array([1, 3, 5,
+                                    6, 7, 8,
+                                    10, 11, 12]).reshape(1, 3, 3)
+        self.assertArrayEqual(result.data,
+                              expected_wxcode)
+
+    def test_lightning(self):
+        """Test process returns right values if all lightning. """
+        plugin = WeatherSymbols()
+        data_lightning = np.array([1.0, 1.0, 1.0, 1.0, 1.0, 1.0,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many positional arguments for method call](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4535748)